Sources/SpeziLLM/SpeziLLM.docc/SpeziLLM.md

```diff
@@ -90,78 +74,92 @@ class LocalLLMAppDelegate: SpeziAppDelegate {
 
 #### Usage
 
-The code section below showcases a complete code example on how to use the ``LLMRunner`` in combination with a `LLMLocal` (locally executed Language Model) from the [SpeziLLMLocal](https://swiftpackageindex.com/stanfordspezi/spezillm/documentation/spezillmlocal) target.
+The code section below showcases a complete, bare-bone code example on how to use the ``LLMRunner`` with the ``LLMSchema``.
+The example is structured as a SwiftUI `View` with a `Button` to trigger LLM inference via the ``LLMMockSchema``. The generated output stream is displayed in a `Text` field.
 
 ```swift
-import SpeziLLMLocal
-// ...
+struct LLMDemoView: View {
+    // The runner responsible for executing the LLM.
+    @Environment(LLMRunner.self) var runner: LLMRunner
 
-struct LocalLLMChatView: View {
-   // The runner responsible for executing the local LLM.
-   @Environment(LLMRunner.self) var runner: LLMRunner
+    // The LLM in execution, as defined by the ``LLMSchema``.
+    @State var llmSession: LLMMockSession?
+    @State var responseText = ""
 
-   // The locally executed LLM
-   @State var model: LLMLocal = .init(
-        modelPath: ...
-   )
-   @State var responseText: String
+    var body: some View {
+        VStack {
+            Button {
+                Task {
+                    try await executePrompt(prompt: "Hello LLM!")
+                }
+            } label: {
+                Text("Start LLM inference")
+            }
 
```

Comment:
As with the other documentation; it might make sense to add some UI management here while the session is not created. A progress indicator might also be fine. Disabling the button is probably easier.